src/app/interface-form/bet-button/bet-button.component.css

```diff
@@ -0,0 +1,21 @@
+.btn-body {
+  margin-top: 12px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.bet-button {
+  background: var(--success);
+  height: 58px;
+  width: 60%;
+  border-radius: var(--main-border-radius);
+  font-size: var(--font-header-size);
+  font-weight: var(--font-header-weight);
+  color: var(--text-color);
```

Comment:
Give this var more understandable name. What color is this?

Reply:
This color names is taken from the design in Figma and the project documentation.
Should i change it?